### PR TITLE
task: release prep for 0.20.0

### PR DESCRIPTION
**Pull request by gssbzn**
Changelog and version bump

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [v0.20.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.20.0) (December 16, 2022)
+
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.19.0...v0.20.0)
+
+**Merged pull requests:**
+
+- Added missing fields 'Links', 'CopySettings' and 'DeleteCopiedBackups… [\#332](https://github.com/mongodb/go-client-mongodb-atlas/pull/332) ([sangramPeerI](https://github.com/sangramPeerI))
```
Comment (colm-quinn) [inline CHANGELOG.md:9]:
```suggestion
- Added missing fields 'Links', 'CopySettings' and 'DeleteCopiedBackups' [\#332](https://github.com/mongodb/go-client-mongodb-atlas/pull/332) ([sangramPeerI](https://github.com/sangramPeerI))
```

Think the ... is a typo

**Review by colm-quinn — APPROVED** [review]
LGTM, minor typo fix, if I'm reading the docs correctly

File: CHANGELOG.md (hunk 1 above)
Comment (gssbzn) [reply CHANGELOG.md:9]: autogenerated, it's picked from the commit message

Comment (colm-quinn) [reply CHANGELOG.md:9]: Thanks TIL

**Pull request merged**